src/item/CustomiesItemFactory.php

```diff
@@ -61,28 +66,31 @@ public function getItemTableEntries(): array {
 	/**
 	 * Registers the item to the item factory and assigns it an ID. It also updates the required mappings and stores the
 	 * item components if present.
+	 * CreativeInventoryInfo not completely implemented yet.
```

Comment:
What is the point of adding this comment? Unimplemented things will not be pushed either way, please add proper documentation instead.

Reply:
This is a comment I forgot to remove this one

Reply:
This is a comment I forgot to remove this one